static/presentations/README.md

```diff
@@ -10,5 +10,5 @@
 | 2017-12-12 | [Vladimir Simek](https://www.linkedin.com/in/vsimek/)      | [Best of reInvent 2017](2017-12-12/2017-12-12-Vladimir_Simek-Best_of_reInvent_2017.pdf)                        |
 | 2017-12-12 | [Roman Pichlik](https://www.linkedin.com/in/romanpichlik/) | [AWS at Zonky](2017-12-12/2017-12-12-Roman_Pichlik-AWS_at_Zonky.pdf)                                           |
 | 2017-12-12 | [Marko Tomic](https://www.linkedin.com/in/tomicmarko/)     | [Serverless Code Deployments in AWS](2017-12-12/2017-12-12-Marko_Tomic-Serverless_Code_Deployments_in_AWS.pdf) |
-
+| 2018-03-07 | [Jiri Psota](https://www.linkedin.com/in/jpsota/)      | [Yieldigo AWS use cases](2018-03-07-Jiri_Psota-Yieldigo_AWS_use_cases.pdf) |
```

Comment:
The link to PDf presentation is not "valid", you should use it with directory like: `2018-03-07/2018-03-07-Jiri_Psota-Yieldigo_AWS_use_cases.pdf`